src/language-server/sql-meta.ts

```diff
@@ -0,0 +1,9 @@
+export type SqlTypeDefinitions = Record<string, SqlTypeDefinition>;
```

Comment:
Please add copyright header to new files.

Reply:
Done